docs/setup/independent/install-kubeadm.md

```diff
@@ -23,32 +24,32 @@ see the [Using kubeadm to Create a Cluster](/docs/setup/independent/create-clust
 * 2 GB or more of RAM per machine (any less will leave little room for your apps)
 * 2 CPUs or more 
 * Full network connectivity between all machines in the cluster (public or private network is fine)
-* Unique hostname, MAC address, and product_uuid for every node
-* Certain ports are open on your machines. See the section below for more details
+* Unique hostname, MAC address, and product_uuid for every node. See the section below for more details.
```

Comment:
can we make "section below" an anchor link?

Reply:
done, thanks